### PR TITLE
EBODAC-959 - DEV - TASK - Maintenance - Create a readme.md file for github

### PR DESCRIPTION
**Pull request by rubenmartin**
Added a readme.md based on the requirements from Motech

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: ebodac/src/main/java/org/motechproject/ebodac/template/PdfBasicTemplate.java [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
 public abstract class PdfBasicTemplate {
 
-    public static final Font TABLE_FONT = new Font(Font.FontFamily.HELVETICA, 8);
-    public static final Font HEADER_FONT = new Font(Font.FontFamily.HELVETICA, 9, Font.BOLD);
+    public static final Font TABLE_FONT = new Font(Font.FontFamily.HELVETICA, 6);
```
Comment (pmuchowski) [inline ebodac/src/main/java/org/motechproject/ebodac/template/PdfBasicTemplate.java:19]: @rubenmartin why did you added this changes?

Comment (rubenmartin) [inline ebodac/src/main/java/org/motechproject/ebodac/template/PdfBasicTemplate.java:19]: I do not understand how is possible , when I merged  it was only the readme.md one file, and now appears more, I do not get it, could you just merge the readme.md ? the rest is not needed , those are old changes we discussed that were not needed, we just need the readme.md, please just merge the readme.md.

**Pull request closed without merging**